mindbender/maya/pythonpath/userSetup.py

```diff
@@ -5,46 +5,26 @@
 import os
 
 
-def mayafpsconverter(Sfps):
-    condition = 0
-    if Sfps is None:
-        return ""
-    if Sfps == "":
-        condition = 1
-        return Sfps
-    if Sfps == "15":
-        condition = 1
-        return "game"
-    if Sfps == "24":
-        condition = 1
-        return "film"
-    if Sfps == "25":
-        condition = 1
-        return "pal"
-    if Sfps == "30":
-        condition = 1
-        return "ntsc"
-    if Sfps == "48":
-        condition = 1
-        return "show"
-    if Sfps == "50":
-        condition = 1
-        return "palf"
-    if Sfps == "60":
-        condition = 1
-        return "ntscf"
+def mayafpsconverter(fps):
     ERRORSTRING = "MINDBENDER_FPS has bad value in the bat file"
-    if str(Sfps).isdigit() is False:
+    if str(fps).isdigit() is False:
```

Comment:
isdigit() checks if there is a letter character in there.
it will accept "12"
but not "1a2"